camel/toolkits/openai_image_toolkit.py

```diff
@@ -123,11 +123,14 @@ def image_to_base64(self, image: Image.Image) -> str:
             logger.error(error_msg)
             return error_msg
 
-    def get_dalle_img(self, prompt: str, image_dir: str = "img") -> str:
-        r"""Generate an image using OpenAI's DALL-E model.
+    def get_img(
```

Comment:
How about renaming it to `generate_image`:
```suggestion
    def generate_image(
```